### ISSUE TITLE
Problem with habitat/navmesh?

### ISSUE THREAD

**Issue by dhruvbatra** [facebookresearch/Replica-Dataset#20]
When apartment0 is viewed with habitat-sim/build/viewer, the agent cannot cross a particular point on the stairs. 

See attached videos -- 
apartment0_floor1: where the agent is spawned on floor1 and gets stuck trying to go up the stairs, and 
apartment0_floor2: where the agent is spawned on floor2 and gets stuck trying to go down. 

Printout on the screen says "distance to closest obstacle = 0". I suspect the navmesh is off?

![apartment0_floor1](https://user-images.githubusercontent.com/2941091/60390437-93a97a00-9a8b-11e9-9c35-1a0cfd4e9473.gif)
![apartment0_floor2](https://user-images.githubusercontent.com/2941091/60390441-a1f79600-9a8b-11e9-834c-250f3903043b.gif)

**Comment by jstraub** [comment]
@erikwijmans can you take a look please?

**Comment by erikwijmans** [comment]
Yep.  We probably just need to change the max-step height of the agent (that is/was just set heuristically)

**Comment by jstraub** [comment]
@erikwijmans on export? Or in in the habitat sim?

**Comment by erikwijmans** [comment]
On export

**Comment by aclegg3** [comment]
This was fixed by setting `agentHeight = 1.4` for navmesh generation.  This is either a very low ceiling in the staircase or there are some approximations being made with the geometry or its navmesh treatment which made this spot an edge case.

**Comment by feji02** [comment]
Why does the floor of the scene and the agent trajectory appear vertical when opening the mesh.ply of Replica in Habitat Sim? This abnormal issue is shown in the figure below. In contrast, everything works correctly and the agent fits perfectly on the ground when loading mesh_semantic.ply.

<img width="1707" height="1280" alt="Image" src="https://github.com/user-attachments/assets/016f4226-45d8-4705-8bf9-ef07a634904c" />
